### PR TITLE
[Merged by Bors] - chore: long lines in tests

### PR DESCRIPTION
**Pull request by adomani**
Hopefully, this is the last adaptation to the long line linter: it now reached the tests, so this should really be everything!

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: test/fun_prop.lean [hunk 1]
```diff
@@ -44,7 +45,8 @@ attribute [fun_prop]
   measurable_pi_lambda
 
 /-!
-Measurability also behaves nicely with respect to taking products. Let's mark the product constructor
+Measurability also behaves nicely with respect to taking products.
+Let's mark the product constructor
```
Comment (grunweg) [inline test/fun_prop.lean:49]:
```suggestion
Let's mark the product constructor.

File: test/fun_prop.lean [hunk 2]
```diff
@@ -59,7 +61,8 @@ or
 ```
 Measurable.fst : Measurable f → Measurable fun x => Prod.fst (f x)
 ```
-Tactic `fun_prop` can work with both versions and it should be sufficient to provide just one of them.
+Tactic `fun_prop` can work with both versions and it should be sufficient to provide just one of
```
Comment (grunweg) [inline test/fun_prop.lean:64]: How about
"The tactic `fun_prop` can work with both versions; it should be sufficient to provide just one of them."?

**Review by grunweg — COMMENTED** [review]
Looks good also, thanks! Two small nit-picks, which I consider optional.
maintainer merge

File: test/fun_prop.lean (hunk 2 above)
Comment (adomani) [reply test/fun_prop.lean]: Thanks: changed!

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**